homework-1/src/Point.ts

```diff
@@ -0,0 +1,34 @@
+export class Point {
+    public x: number;
+    public y: number;
+
+    constructor();
+    constructor(x?: number, y?: number) {
```

Comment:
According to the Hometask "x" and "y" should be obligatory for this overloading

Reply:
Fixed